README.md

```diff
@@ -12,35 +12,37 @@ It will automatically update your packages and configure these packages to secur
 - [NTP](https://ubuntu.com/server/docs/network-ntp)
 
 The playbook also:
-- Remove [Snap](https://snapcraft.io/).
-- Disable ssh password login.
+
+- Removes [Snap](https://snapcraft.io/).
+- Disables ssh password login.
 
 ## Getting Started
 
 Clone the repo:
-```bash
-$ git clone git@github.com:guillaumebriday/kamal-ansible-manager.git
-$ cd kamal-ansible-manager
-```
 
-Copy the inventory example file:
 ```bash
-$ cp hosts.ini.example hosts.ini
+git clone git@github.com:guillaumebriday/kamal-ansible-manager.git
+cd kamal-ansible-manager
 ```
 
-Update the `<host1>` with your server's IP address (you can have multiple servers):
+Copy the example files:
+
 ```bash
-$ vim hosts.ini
+cp hosts.ini.example hosts.ini
+cp ansible.cfg.example ansible.cfg
 ```
 
+Update the `<host1>` in `hosts.ini` with your server's IP address (you can have multiple servers):
+
 Run the playbook:
+
 ```bash
-$ ANSIBLE_HOST_KEY_CHECKING=False ansible-playbook -i hosts.ini playbook.yml
+ANSIBLE_HOST_KEY_CHECKING=False ansible-playbook -i hosts.ini playbook.yml
```

Comment:
tbf the inline env var and hosts arg aren't necessary with `ansible.cfg` left for posterity / if end users don't use the `ansible.cfg.example`